saml_idp.gemspec

```diff
@@ -47,6 +47,7 @@ section of the README.
   s.add_dependency('uuid', '>= 2.3')
   s.add_dependency('builder', '>= 3.0')
   s.add_dependency('nokogiri', '>= 1.6.2')
+  s.add_dependency('xmlenc', '>= 0.6.4')
```

Comment:
@TimFung93 do you mind if I ask change min version ">= 0.7.1" for your changes?

Reply:
done